### DISCUSSION TITLE
Turn restriction "except bicycle" getting ignored

### DISCUSSION THREAD

**Discussion by pinkdroyd** [osmandapp/OsmAnd#17503] [Maps Related]
OsmAnd routing for bicycle complies with turn restrictions even if the relation in OSM includes the `except=bicycle` tag, so it does not concern cyclists.
Here are two examples: the first is a `straight_ahead_only`, the second one is `right_turn_only`. I remember the second example working properly (even though I don't know when a change occured).

- https://www.openstreetmap.org/relation/15489136 (https://osmand.net/map?start=49.026466%2C12.070635&end=49.025658%2C12.071604&mode=bicycle#19/49.02572/12.07261) (right turn is allowed for cyclists)
- https://www.openstreetmap.org/relation/125109 (https://osmand.net/map?start=49.013238%2C12.103095&end=49.012587%2C12.101575&mode=bicycle#19/49.01307/12.10308) (left turn is allowed for cyclists)

**Comment by martinrieder** [comment]
Possible duplicate of #3882 ?
